doc/source/examples/analysis/distances_and_contacts/distances_between_atomgroups.ipynb

```diff
@@ -76,7 +76,17 @@
      "shell.execute_reply": "2021-05-19T05:57:16.889497Z"
```

Comment:
<p>Add a filterwarning for this too?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MDAnalysis/UserGuide/pull/241/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Fixed.</p>